src/api/public/apidocs-new/paths/build_project_repository_arch_package_log.yaml

```diff
@@ -0,0 +1,76 @@
+get:
+  description: This endpoint returns the log file for a specific build artifact
+  operationId: getBuildProjectRepositoryArchPackageLog
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
+      name: project
+      schema:
+        type: string
+      example: "home:Admin"
+      required: true
+      description: "The project name"
+    - in: path
+      name: repository
+      schema:
+        type: string
+      example: "openSUSE_Tumbleweed"
+      required: true
+      description: "The repository name"
+    - in: path
+      name: architecture
+      schema:
+        type: string
+      example: "i586"
+      required: true
+      description: "The architecture short name"
+    - in: path
+      name: package
+      schema:
+        type: string
+      example: "ctris"
+      required: true
+      description: "The package name"
```

Comment:
There are 3 open PRs that define some combination of these parameters. Does it make sense to move them to their own files and add references to them whenever we need them?

I commented the same in PR #10976 and tried to do what I'm proposing in PR  #10975.
We need to agree regarding the names of the parameters: architecture VS architecture_name, for example.